src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+/* eslint-disable */
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  let activeTab = tabs.find(tab => tab.id === activeTabId);
+
+  if (!activeTab) {
+    activeTabId = tabs[0].id;
+    activeTab = tabs[0];
+  }
```

Comment:
Using let in the global scope of components is not good practice for readability and code support. 
Please, try to rewrite without using let.